src/classes/Forms.ts

```diff
@@ -73,6 +73,23 @@ export default class Forms extends OneBlinkAPI {
    *
    * @param parameters An object containing all parameters to be passed to the
    *   function
+   * @returns
+   *
+   *   ## Role Permissions Required
+   *
+   *   ### For Private Forms
+   *
+   *   Forms: `Manager` or `Read Only`
```

Comment:
Why do you need forms here?

Reply:
Line 48-52 of API in `get-form.ts`. This checks if the user should have access to the Form, for a key it will check the Forms permissions

Reply:
Ah, i see we are fetching the form as part of the endpoint. I did not think we need to do that. Moving on 😄